homeassistant/components/home_connect/api.py

```diff
@@ -93,15 +93,15 @@ def initialize(self):
         try:
             self.appliance.get_status()
         except (HomeConnectError, ValueError):
-            _LOGGER.debug("Unable to fetch appliance status. Probably offline")
+            _LOGGER.debug("Unable to fetch appliance status. Probably offline.")
```

Comment:
There are still changes that aren't addressed from my comments.

Reply:
Sorry, some have been forgotten.